$NetBSD: patch-ba,v 1.2 2010/07/15 20:23:10 dholland Exp $

Patch from upstream to stop emacs from trying to read stdin during the
build. From Makoto Fujiwara in PR 43580.

This patch should become unnecessary with the next update.

--- doc/wl.texi.orig	2005-02-25 15:05:48.000000000 +0000
+++ doc/wl.texi
@@ -4329,7 +4329,7 @@
 
 @item wl-summary-resend-use-cache
 @vindex wl-summary-resend-use-cache
-The initial setting is @code{nil}¡£If non-nil, messages are resend
+The initial setting is @code{nil}.  If non-nil, messages are resend
 using cache even in the offline status. Note that if you use cache,
 the message identity is not guaranteed.
 
